pkg/scheduler/plugin.go

```diff
@@ -157,7 +156,7 @@ func (b *binder) bind(ctx api.Context, binding *api.Binding, task *PodTask) (err
 
 	if err = b.prepareTaskForLaunch(ctx, binding.Host, task); err == nil {
 		log.V(2).Infof("Attempting to bind %v to %v", binding.PodID, binding.Host)
-		if err = b.client.Post().Namespace(api.Namespace(ctx)).Path("bindings").Body(binding).Do().Error(); err == nil {
+		if err = b.client.Post().Namespace(api.Namespace(ctx)).Resource("bindings").Body(binding).Do().Error(); err == nil {
```

Comment:
I see your point. This line is basically copy/pasted from k8s code. Upon further inspection, k8s implements a nice error propagation pattern in Request objects:

https://github.com/GoogleCloudPlatform/kubernetes/blob/release-0.8/pkg/client/request.go

I'm probably going to leave this as-is.